x-pack/plugins/task_manager/server/lib/calculate_health_status.ts

```diff
@@ -9,32 +9,42 @@ import { isString } from 'lodash';
 import { JsonValue } from '@kbn/utility-types';
 import { HealthStatus, RawMonitoringStats } from '../monitoring';
 import { TaskManagerConfig } from '../config';
+import { Logger } from '../../../../../src/core/server';
 
 export function calculateHealthStatus(
   summarizedStats: RawMonitoringStats,
-  config: TaskManagerConfig
+  config: TaskManagerConfig,
+  logger: Logger
 ): HealthStatus {
   const now = Date.now();
 
-  // if "hot" health stats are any more stale than monitored_stats_required_freshness (pollInterval +1s buffer by default)
```

Comment:
Note to fix comment: https://github.com/elastic/kibana/pull/109741/files#diff-6d40601cf6359ce2293adca1d86f0f6e25f026c15edf8f013bc349ca23f2588cR60

>   // if "hot" health stats are any more stale than monitored_stats_required_freshness (pollInterval +1s buffer by default)
>   // consider the system unhealthy

Reply:
Heh, ya, this is confusing.  This particular setting, and I think the equivalent cold one, are used in two different ways:

- a throttle in the `heath.ts` file, which indicates how often it should update the status - which ends up being pollInterval + 1s by default; kinda matching the 5 second interval I've seen in logs elsewhere (add 1s for various latencies).

- a measure of whether the stats are too old

Previously, the same value, pollInterval + 1s, was used for both.  In this PR, we're changing the second usage - determining if the stats are too old, by multiplying that pollInterval + 1s value by a constant (3, right now).

Ideally, these should be two different config values, and using the same value for both purposes felt like it was going to be too close for comfort.  Changing it to `* 3` on the slop-factor (checking for staleness), seemed to work out great in a scenario of temporarily overloading Kibana with rules, then deleting them.  Went degraded when overloaded, a minute or so after deleting the rules, everything went back to available.

So as far as I know, the comment in `health.ts` is still correct, because we haven't change the actual config value, it's still operating the same way it used to.

Reply:
Oh I see, this is a miss on my part, thanks!